### DISCUSSION TITLE
Live Wallpaper enhancements to reflect season, wind speed etc.

### DISCUSSION THREAD

**Discussion by tjsuominen** [breezy-weather/breezy-weather#876] [Ideas]
Currently the live wallpaper shows clouds, sun(shine) on the top of the screen and rain animation across the display 👍

There are certain aspects that would also be great visualizing on the wallpaper to further elaborate ongoing weather: landscape showing season (winter, spring, summer, autumn) and wind speed to mention two most obvious. Also visibility would be one.

That (landscape) could be *optionally* shown at the bottom of the screen as very simple forest, city silhouette, beach or similar landscape. There could be visuals then expressing the year seasoning of tree(s), wind speed (eg. how fast objects fly; say leaves, umbrellas, trees bend).

Would it make sense to use the bottom of the screen for visuals (season, landscape) and additional animations (wind speed, fog/visibility)?

How does this sound?

And yes, as optional (for the bottom landscapes) AND as very simple in aesthetics as the top of the screen.

Just for thoughts.

Any takers?

**Comment by papjul** [comment]
For seasons, that would be way too complicated: [too many different climates around the world](https://en.m.wikipedia.org/wiki/Köppen_climate_classification) resulting in different expressing of seasons, different type of trees around the world (eucalyptus, pines, cactus, palm trees, just to name a few), different type of soils, etc.

However, for the wind speed, it was suggested to me, however I can't find the issue anymore, so maybe it was never reported. Someone showed me an app that already does that but I don't remember which. I also have no skill for animations, so someone else will have to contribute it.

**Reply by tjsuominen** [reply]
See discussions #313 (for wind animation) - the App is Today Weather

**Comment by tjsuominen** [comment]
The point was not to reflect every type of "landscape", just the one you want to design.
